### DISCUSSION TITLE
Numerous NullPointerExceptions in AntMedia server logs (v2.6.2)

### DISCUSSION THREAD

**Discussion by dalkofjac** [ant-media/Ant-Media-Server#5309] [General]
Hi all!

To continue the discussion started here: https://community.antmedia.io/topic/1422-numerous-nullpointerexceptions-in-antmedia-server-logs-v243, I'd just like to point out that the issue is still persistent in the latest AntMedia Enterprise version 2.6.2.

A quick screenshot from the logs (as the issue is the same as in the logs given in the community ticket):
![image](https://github.com/ant-media/Ant-Media-Server/assets/17647730/55b924de-2317-4fab-a110-77a0db04c733)

It seems the exceptions happen in a pattern, every day in 12 AM, around 100 of them at a time.

Is there any follow up whether that's been or will be taken care of?

Thanks!

**Comment by mekya** [comment]
Hi @dalkofjac,
I've created the issue from this discussion to schedule.

Here is the issue
https://github.com/ant-media/Ant-Media-Server/issues/5350

**Comment by USAMAWIZARD** [comment]
@dalkofjac Is there any reproduce scenario for this issue or is it happening randomly for your case  ?

**Reply by dalkofjac** [reply]
Hi @USAMAWIZARD,

It's happening randomly so to say, the only pattern I've seen is the fact that it seems to be happening every day in 12 AM, around 100 of them at a time. Other than that, I've seen it also happen here and there not in that time span. The version of AntMedia server doesn't matter, we've noticed it in every version we've used so far (every official version up to now).

Also, we're using AntMedia server in an Azure Kubernetes setup, it's probably important to note that.

**Reply by mekya** [reply]
Hi @dalkofjac 

I think it's related to internal communication of the nodes in the Kubernetes setup. 

I think this PR https://github.com/ant-media/Ant-Media-Server/pull/5414 will fix this issue as well. 

We plan to merge this PR, maxAudioTrack(I think it's also related to you) for the next release. We plan to have next release in this week. 

FYI

Thank you for followup @USAMAWIZARD
